upgrade_test.py

```diff
@@ -605,7 +605,7 @@ def _update_scylla_yaml_on_node(node_to_update: BaseNode, updates: dict):
         with node_to_update.remote_scylla_yaml() as scylla_yaml:
             scylla_yaml.update(updates)
 
-    def test_rolling_upgrade(self):  # pylint: disable=too-many-locals,too-many-statements
```

Comment:
The `pylint: disable=too-many-locals,too-many-statements` becomes redundant and must be removed.